### PR TITLE
No upcoming events state for events list widget

### PR DESCRIPTION
**Pull request by paulmskim**
**Ticket:** [TEC-3622]

This PR adds the messages component and styles to display the no upcoming events message in the events list widget.

Waiting on https://github.com/moderntribe/the-events-calendar/pull/3318 and https://github.com/moderntribe/the-events-calendar/pull/3316 to be merged.

**Artifact:**

image

[TEC-3622]: https://moderntribe.atlassian.net/browse/TEC-3622

### CODE REVIEW

File: src/Tribe/Views/V2/Views/Widgets/Widget_List_View.php [hunk 1]
```diff
@@ -86,10 +86,11 @@ protected function setup_template_vars() {
 		$template_vars = parent::setup_template_vars();
 
 		// Here update, add and remove from the default template vars.
-		$template_vars['view_more_link']     = tribe_get_events_link();
-		$template_vars['widget_title']       = $this->context->get( 'widget_title' );
-		$template_vars['no_upcoming_events'] = $this->context->get( 'no_upcoming_events' );
-		$template_vars['show_latest_past']   = false;
+		$template_vars['events'] = [];
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Views/Widgets/Widget_List_View.php:89]: :warning: Equals sign not aligned with surrounding assignments; expected 21 spaces but found 1 space

<sub>hash: ea5b4eac00ff4d35594cd826bced5afb</sub>

**Review by tr1b0t — COMMENTED** [review]
:warning: 1 warning(s)

File: src/Tribe/Views/V2/Views/Widgets/Widget_List_View.php (hunk 1 above)
Comment (tr1b0t) [inline src/Tribe/Views/V2/Views/Widgets/Widget_List_View.php:89]: :warning: Equals sign not aligned with surrounding assignments; expected 21 spaces but found 1 space

<sub>hash: 5a02464b05d5da87e56961f2787df470</sub>

File: src/Tribe/Views/V2/Views/Widgets/Widget_View.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	protected function setup_template_vars() {
+		$template_vars = parent::setup_template_vars();
+
+		$template_vars['container_classes'] = $this->get_html_classes();
+
+		return $template_vars;
+	}
+
+	/**
+	 * {@inheritDoc}
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Views/Widgets/Widget_View.php:38]: :no_entry_sign: Doc comment for parameter "$classes" missing

<sub>hash: e9da3b783c3bf4646ba05e0f21b15100</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 1 error(s)
:warning: 1 warning(s)

**Review by bordoni — APPROVED** [review]

**Review by jesseeproductions — APPROVED** [review]
Looks good. I updated the widget snapshots and they passed.

**Pull request merged**